custom_model_runner/datarobot_drum/drum/perf_testing.py

```diff
@@ -587,40 +605,43 @@ def check_prediction_side_effects(self):
             _, __tempfile_sample = mkstemp(suffix=".csv")
             data_subset.to_csv(__tempfile_sample, index=False)
 
-        if self.target_type == TargetType.BINARY:
-            labels = [self.options.negative_class_label, self.options.positive_class_label]
-        elif self.target_type == TargetType.MULTICLASS:
-            labels = self.options.class_labels
-        else:
-            labels = None
+        labels = self.set_labels(self.target_type, self.options)
 
         with DrumServerRun(
-            self.target_type.value,
-            labels,
-            self.options.code_dir,
+            self.target_type.value, labels, self.options.code_dir, verbose=False
         ) as run:
             response_key = (
```

Comment:
looks like `response_key` is only used in `else` condition, so there is no need for this code

Reply:
this ended up getting called for transform and predict, so I think we still need this